src/snowcli/utils.py

```diff
@@ -582,3 +582,15 @@ def create_project_template(template_name: str):
         f"{os.getcwd()}",
         dirs_exist_ok=True,
     )
+
+
+def sql_to_python_return_type_mapper(resource_return_type: str) -> str:
+
+    mapping = {
+        "number(38,0)": "int",
+        "timestamp_ntz(9)": "datetime",
+        "timestamp_tz(9)": "datetime",
+        "varchar(16777216)": "string",
+    }
```

Comment:
Can we add link to the source of this information for future references?

Reply:
Added